src/_guides/language/language-tour.md

```diff
@@ -4241,6 +4240,25 @@ Keep in mind the following when you use deferred loading:
   using <code>deferred as <em>namespace</em></code>.
   The `loadLibrary()` function returns a [`Future`](/guides/libraries/library-tour#future).
 
+#### The `library` directive {#library-directive}
+
+The only reason to use a library directive is
+to attach library-level [doc comments][] or [metadata annotations][].
+Library-level comments and annotations need to prepend
+a `library` declaration at the start of a file.
```

Comment:
Reading this again, saying "only reason" makes this seem like readers may know other incorrect reasons, but this is a learning material. These two sentences also seem like they mostly cover the same thing. Could we just directly state what the library directive is for, following other areas in the language tour?

```suggestion
To specify library-level [doc comments][] or [metadata annotations][],
attach them to a `library` declaration at the start of the file.
```

Reply:
This makes the next sentence, "There’s also no reason to name a library using the directive; in fact, it’s [discouraged](http://0.0.0.0:4000/guides/language/effective-dart/style#dont-explicitly-name-libraries)." pretty weird. Sticking with your point that this is learning material, there's no need to say "Don't name libraries" here. I'm going to remove it.